OrbitClientModel/CaptureDeserializer.cpp

```diff
@@ -148,6 +155,28 @@ void LoadCaptureInfo(const CaptureInfo& capture_info, CaptureListener* capture_l
     capture_listener->OnCallstackEvent(std::move(callstack_event));
   }
 
+  for (const orbit_client_protos::TracepointInfo& tracepoint_info :
+       capture_info.tracepoint_infos()) {
+    if (*cancellation_requested) {
+      capture_listener->OnCaptureCancelled();
+      return;
+    }
+    orbit_grpc_protos::TracepointInfo tracepoint_info_translated;
+    tracepoint_info_translated.set_category(tracepoint_info.category());
+    tracepoint_info_translated.set_name(tracepoint_info.name());
+    capture_listener->OnUniqueTracepointInfo(tracepoint_info.tracepoint_info_key(),
```

Comment:
As we have the selected tracepoints in capture_data already, do we actually need to the OnUniqueTracepointInfo part at all? could't we use the ones from selected tracepoint to compute this mapping?

This is more of a general question and should not be addressed in this PR.